Doc/using/cmdline.rst

```diff
@@ -994,3 +1000,110 @@ Debug-mode variables
+to be in :file:`lib`. Note that the expected zip archive location is added to the
+module search path even if the archive does not exist. If no archive was found,
+Python on Windows will continue the search for ``prefix`` by looking for :file:`Lib\\os.py`.
+Python on Unix will look for :file:`lib/python{majorversion}.{minorversion}/os.py`
+(``lib/python3.11/os.py``). On Windows ``prefix`` and ``exec_prefix`` are the same,
+however on other platforms :file:`lib/python{majorversion}.{minorversion}/lib-dynload`
+(``lib/python3.11/lib-dynload``) is searched for and used as an anchor for
+``exec_prefix``.
+
+Once found, ``prefix`` and ``exec_prefix`` may be accessed at :data:`sys.prefix`
+and :data:`sys.exec_prefix`.
+
+Finally, the :mod:`site` module is processed and :file:`site-packages` directories
+are added to the module search path. A common way to customize the search path is
+to create :mod:`sitecustomize` or :mod:`usercustomize` modules as described in
+the :mod:`site` module documentation.
+
+.. note::
+
+   Certain command line options may further affect path calculations.
+   See :ref:`-E <using-on-misc-option-uppercase-e>`, :ref:`-I <using-on-misc-option-uppercase-i>`, :ref:`-s <using-on-misc-option-s>` and :ref:`-S <using-on-misc-option-uppercase-s>` for further details.
+
+Virtual environments
+~~~~~~~~~~~~~~~~~~~~
+
+If Python is run in a virtual environment (as described at :ref:`tut-venv`)
+then ``prefix`` and ``exec_prefix`` are specific to the virtual environment.
+
+If a ``pyvenv.cfg`` file is found alongside the main executable, or in the
+directory one level above the executable, the following variations apply:
+
+* If ``home`` is an absolute path and :envvar:`PYTHONHOME` is not set, this
+  path is used instead of the path to the main executable when deducing ``prefix``
+  and ``exec_prefix``.
+
+_pth files
+~~~~~~~~~~
+
+To completely override :data:`sys.path`, create a ``._pth`` file with the same
+name as the shared library (``python311._pth``) or the executable (``python._pth``)
+and specify one line for each path to add to :data:`sys.path`. The file based on
+the shared library name overrides the one based on the executable, which allows
+paths to be restricted for any program loading the runtime if desired.
+
+When the file exists, all registry and environment variables are ignored,
+isolated mode is enabled, and :mod:`site` is not imported unless one line in the
+file specifies ``import site``. Blank paths and lines starting with ``#`` are
+ignored. Each path may be absolute or relative to the location of the file.
+Import statements other than to ``site`` are not permitted, and arbitrary code
+cannot be specified.
+
+Note that ``.pth`` files (without leading underscore) will be processed normally
+by the :mod:`site` module when ``import site`` has been specified.
+
+Embedded Python
+~~~~~~~~~~~~~~~
+
+If Python is embedded within another application :c:func:`Py_SetPath` can be used to
```

Comment:
Py_SetPath is deprecated. I suggest to document the PyConfig API first.

There is also Py_SetPathEx.

Reply:
Ok, will do